### ISSUE TITLE
Code generation is broken for requests with JSON data in C#, Python and JavaScript

### ISSUE THREAD

**Issue by benthillerkus** [rangav/thunder-client-support#395]
**Describe the bug**
JSON like
```JSON
{
  "fortnite": true
}
```
will be turned into
```
"{\n  \"fortnite\": true\n}"
```
which isn't valid.

**To Reproduce**
Build a request involving JSON data and look at the code being generated.

**Expected behavior**
The code examples should work.

**Platform:** 
 - OS: Windows 10
 - vscode version: 1.62.2
 - extension version: 1.9.1

**Solution:**
Atleast in case of JavaScript the cleanest possible code to be generated in this example would in my opinion be [^1]
```JavaScript
JSON.stringify({
  "fortnite": true
})
```

[^1]: This is more of a feature request, but it would be great if the code examples also automatically put `{{env}}` variables into separate variables that are being put into the JSON, either via format strings or by having the JSON be serialized like with `JSON.stringify`.

**Comment by rangav** [comment]
Hi @benthillerkus 

Have tested when the code generated is working or not?

**Comment by rangav** [comment]
This feature is now implemented in v1.11.0
https://github.com/rangav/thunder-client-support/releases/tag/v1.11.0

please let me know if any feedback
